python/py-blosc2/Portfile

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-blosc2
+version             2.2.0
+revision            0
+categories-append   devel
+license             BSD
+
+python.versions     37 38 39 310 311
+
+maintainers         nomaintainer
+
+description         Python package that wraps the Blosc2 compressor
+
+long_description    Blosc (http://www.blosc.org) is a high performance \
+                    compressor optimized for binary data.  It has been \
+                    designed to transmit data to the processor cache faster \
+                    than the traditional, non-compressed, direct memory fetch \
+                    approach via a memcpy() OS call. \
+                    \
+                    Blosc works well for compressing numerical arrays that \
+                    contains data with relatively low entropy, like sparse \
+                    data, time series, grids with regular-spaced values, etc. \
+                    \
+                    C-Blosc2 is the new major version of C-Blosc, and \
+                    is backward compatible with both the C-Blosc1 API \
+                    and its in-memory format. Python-Blosc2 is a \
+                    Python package that wraps C-Blosc2, the newest \
+                    version of the Blosc compressor.
+
+homepage            https://www.blosc.org/python-blosc2/python-blosc2.html
+
+checksums           rmd160  e23ddcafb6a0e2d6da0fb4377e81d7c6227178d8 \
+                    sha256  4cf3893fe85da4bd899ecf7a457fd09b2212fad256182100108de7a92b88aa87 \
+                    size    4135248
+
+if {${name} ne ${subport}} {
+    # Exception: py-cpuinfo currently only works on X86 and some ARM CPUs.
+    #supported_archs     i386 x86_64
+
+    depends_build-append \
+                        port:ninja \
+                        port:py${python.version}-numpy \
+                        port:py${python.version}-scikit-build
+    depends_lib-append  path:bin/cmake:cmake \
+                        port:py${python.version}-cpuinfo \
+                        path:bin/cython:py${python.version}-cython \
+                        port:py${python.version}-msgpack \
+                        port:py${python.version}-ndindex \
+                        port:py${python.version}-rich
```

Comment:
the dependencies don't look completely correct. From what I gather how upstream lists thing is that:
1. numpy is also a runtime dependency
2. cython and cmake should only be a build dependency
3. setuptools is also listed as build dependency

Reply:
Thanks for checking this. Dependencies should now be correct.